### PR TITLE
Update realibility of ci and docker-compose

### PR DESCRIPTION
**Pull request by Gauvino**

### CODE REVIEW

**Review by CyferShepard — CHANGES_REQUESTED** [review]
Hi @Gauvino , thanks for the changes, everything looks great so far. Only thing I need changed is in the compose file. The postgres image has to be `postgres:15.2` as per the docker hub page, `latest` postgres tag breaks existing installs

File: docker-compose.yml [hunk 1]
```diff
@@ -1,36 +1,43 @@
...
-    restart: unless-stopped
+      
 networks:
   default:
+
+volumes:
+  postgres-data:
+  jellystat-backup-data:
+
 logging:
```
Comment (Gauvino) [inline docker-compose.yml:39]: Does this I really relevant to keep?

**Comment by Gauvino** [comment]
> Hi @Gauvino , thanks for the changes, everything looks great so far. Only thing I need changed is in the compose file. The postgres image has to be `postgres:15.2` as per the docker hub page, `latest` postgres tag breaks existing installs

Changed it !

File: docker-compose.yml (hunk 1 above)
Comment (CyferShepard) [reply docker-compose.yml:39]: Hey, yea it is, without the postgres one, the data gets wiped whenever the container is restarted, and the jellystat one is needed to perform backups

**Pull request merged**